CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
+- Pixel event triggered when a filter is selected
```

Comment:
```suggestion
### Added
- Pixel event triggered when a filter is changed

```

Reply:
Done in last commit (required operations)